lib/user-handler.js

```diff
@@ -1002,13 +1059,19 @@ class UserHandler {
                     // ignore
                 }
 
-                return await authSuccess({
+                const authResponse = {
                     user: userData._id,
                     username: userData.username,
                     scope: requiredScope,
                     asp: asp._id.toString(),
                     require2fa: false // application scope never requires 2FA
-                });
+                };
+
+                if (isPasswordPwned) {
+                    authResponse.passwordPwned = isPasswordPwned;
```

Comment:
Should this also be added to the authentication response schema? https://docs.wildduck.email/docs/wildduck-api/authenticate#responses